cmd/update-notifier.go

```diff
@@ -27,6 +27,61 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
+func humanizeDuration(duration time.Duration) (durationStr string) {
```

Comment:
Why not deprecate this and use - https://github.com/dustin/go-humanize/blob/master/times.go